### DISCUSSION TITLE
tooltip 组件如何在显示状态下变更显示位置? (update or forceUpdate)

### DISCUSSION THREAD

**Discussion by imguolao** [IDuxFE/idux#472] [Q&A]
tooltip 和 overlay 都没有暴露 update 方法，让 tooltip 能在显示过程中根据 dom 变化更新显示位置等。

当 tooltip 的 target dom 变更位置，比如从页面右边移到左边，tooltip 只能隐藏后再显示才会从正确的位置出现 (这样会有闪烁)，无法通过 forceUpdate 等方法随着 target dom 一起变换位置。

![无标题](https://user-images.githubusercontent.com/25502024/136887040-99e9ee49-c627-44a3-a143-93e1ff9a96dd.png)

是否有什么方法可以实现该操作？

https://github.com/IDuxFE/idux/blob/78738c2b819beaa5b52e15fc41ba518b99bb05d4/packages/components/_private/overlay/src/Overlay.tsx#L37-L50

**Comment by LaamGinghong** [comment]
@danranVm 可以增加一个update的方法
